tests/rbd/test_scale_group_snap_clone.py

```diff
@@ -0,0 +1,241 @@
+
+            # Verify the group snapshot exists using the rbd group snap ls command.
+            log.info(f"Verify group id for group: {group} snap: {group_snap}")
+            group_ls_kw = {
+                "client": client,
+                "pool": pool,
+                "group": group,
+                "format": "json",
+            }
+            (g_ls_out, _) = group_info(**group_ls_kw)
+            log.info(g_ls_out)
+            g_ls_out = json.loads(g_ls_out)
+            if g_ls_out["group_id"].isalnum():
+                log.info(f"Group id exist for group {group}")
+            else:
+                log.error(f"Group id does not exist for group {group}")
+                return 1
+
+            # Get Snap-id
+            log.info(
+                "Get snap-id of group snapshot to further create clone of image from group snapshot"
+            )
+            snap_group_info = {
+                "client": client,
+                "pool": pool,
+                "group": group,
+                "snap": group_snap,
+                "format": "json",
+            }
+            (snap_g_out, _) = group_snap_info(**snap_group_info)
+            log.info(snap_g_out)
+            snap_g_out = json.loads(snap_g_out)
+            snap_id_dict = {}
+            for image, image_config in pool_config.items():
+                for i in snap_g_out["images"]:
+                    if i["image_name"] == image:
+                        snap_id_dict[image] = i["snap_id"]
+            log.info(f"{snap_id_dict}")
+
+            for image, image_config in pool_config.items():
+                # Clone the group snapshot for all 100+ images using the rbd clone --snap-id option
+                log.info(
+                    f"Clone the image using group snapshot group: {group} snap: {group_snap}"
+                )
+                clone = image + "_clone"
+                clone_create_kw = {
+                    "source-snap-spec": pool + "/" + image,
+                    "dest-image-spec": pool + "/" + clone,
+                    "snap-id": snap_id_dict[image],
+                    "rbd-default-clone-format": "2",
+                }
+                rbd1.clone(**clone_create_kw)
+
+                # Verify the cloned images exist in the pool and check their parent image using rbd info in a loop.
+                log.info("Verify the cloned images exist in the pool")
+                info_spec = {"image-or-snap-spec": f"{pool}/{clone}", "format": "json"}
+                out, err = rbd1.info(**info_spec)
+                if err:
+                    log.error(f"Error while fetching info for image {clone}")
+                    return 1
+                out_json = json.loads(out)
+                if out_json["name"] != clone:
+                    log.error(f"Clone image {clone} does not exist")
+                    return 1
+
+            # Map the cloned images as new block disks and run fio using rbd map in a loop.
```

Comment:
`run fio using rbd map` sentence looks confusing it should be `run fio on mapped disk`

Reply:
Done